quickwit/quickwit-metastore/migrations/postgresql/12_create-shards.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TYPE SHARDSTATE AS ENUM ('SHARD_STATE_UNSPECIFIED', 'SHARD_STATE_OPEN', 'SHARD_STATE_UNAVAILABLE', 'SHARD_STATE_CLOSED');
+
+CREATE TABLE IF NOT EXISTS shards (
+    index_uid VARCHAR(288) NOT NULL,
```

Comment:
Why `288` and not `282`? (255 + 1 for `:` separator + 26 for ULID)?

Reply:
Yup it should be 282, I mistaked it for 288.